### PR TITLE
Fix small bugs

### PR DESCRIPTION
**Pull request by domakun**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -269,12 +269,13 @@
                 .pipe(
                     throttleTime(6000, asyncScheduler, {leading: true, trailing: true})
                 )
-                .subscribe(({newValue, oldValue}) => {
+                .subscribe(async ({newValue, oldValue}) => {
                     if (newValue && oldValue !== newValue) {
-                        this.Network.switchNode(newValue);
+                        await this.Network.switchNode(newValue);
```
Comment (evias) [inline src/App.vue]: await in App.vue probably needs to be moved

Comment (evias) [reply src/App.vue]: please rebase to what decentraliser has been working on with the App.vue rewrite

File: src/config/constants.ts [hunk 2]
```diff
@@ -157,7 +157,7 @@ export const NETWORK_CONSTANTS = {
     // todo not sure about max value = 2101400 in validator
     MIN_NAMESPACE_DURATION: 30 * 24 * 60 * 60 / networkConfig.targetBlockTime,
     PRIVATE_KEY_LENGTH: 64,
-    NEMESIS_BLOCK_TIMESTAMP: 1459468800,
+    NEMESIS_BLOCK_TIMESTAMP: 1573430400,
```
Comment (evias) [inline src/config/constants.ts]: Why not use the value from `nem2-sdk` in Deadline class?

File: src/core/model/ChainStatus.ts [hunk 3]
```diff
@@ -31,7 +31,9 @@ export class ChainStatus {
             targetBlockTime: networkConfig.targetBlockTime,
             height: 0,
             timestamp: 0,
-            getTimeFromBlockNumber: function() { return null }
+            getTimeFromBlockNumber: function () {
```
Comment (evias) [inline src/core/model/ChainStatus.ts]: why adding this method if returns null?

File: src/App.vue (hunk 1 above)
Comment (domakun) [reply src/App.vue]: fixed

File: src/config/constants.ts (hunk 2 above)
Comment (domakun) [reply src/config/constants.ts]: fixed

File: src/core/model/ChainStatus.ts (hunk 3 above)
Comment (domakun) [reply src/core/model/ChainStatus.ts]: fixed

File: src/views/wallet/wallet-switch/WalletSwitchTs.ts [hunk 4]
```diff
@@ -145,6 +146,22 @@ export class WalletSwitchTs extends Vue {
         this.scrollToActiveWallet()
     }
 
+    deleteWallet(walletToDelete) {
```
[the remark this answers is not in the document]
Comment (domakun) [reply src/views/wallet/wallet-switch/WalletSwitchTs.ts]: fixed

**Pull request merged**